### ISSUE TITLE
Not able to hide chart title in line & pie chart

### ISSUE THREAD

**Issue by ankit-maheshwari** [scanny/python-pptx#302]
Hi,

I am not able to hide chart title with the following code in "line" and "pie" chart.

   ```
from pptx import Presentation
from pptx.chart.data import ChartData
from pptx.enum.chart import XL_CHART_TYPE
from pptx.util import Inches

prs = Presentation()
blank_slide_layout = prs.slide_layouts[6]
slide = prs.slides.add_slide(blank_slide_layout)

chart_data = ChartData()
chart_data.categories =  ['May 01', 'May 02', 'May 03', 'May 04', 'May 05']
chart_data.add_series('Total fans', (100, 120, 130, 140, 150))
chart =slide.shapes.add_chart(XL_CHART_TYPE.LINE_MARKERS,Inches(0.7), Inches(1.42), Inches(9), Inches(2.5), chart_data).chart
#hide chart title
chart.has_title = False
prs.save("line_chart.pptx")

**Comment by jdelight** [comment]
I'm seeing this too `chart.has_title = False` doesn't seem to work.

**Comment by scanny** [comment]
I think this is an unexpected behavior of PowerPoint. In a line chart, I think you'll find it goes away if you add a second series, i.e. it only occurs on single-series charts.

I would start by trying adding a blank chart title and see what you get.

I think if you examine the chart XML you'll find there's no title element present in these cases, rather it's something PowerPoint renders by default for single-value charts, perhaps a bug or at least a mis-feature.

**Comment by jdelight** [comment]
Cool thanks @scanny - seems similar to #293 Currently getting round it by adding a nicely formatted custom title. But you're right it's picking the title of the first series by default.

> I think if you examine the chart XML you'll find there's no title element present in these cases, rather it's something PowerPoint renders by default for single-value charts, perhaps a bug or at least a mis-feature.

That's really interesting - thanks. Good to know it's not just a bug with Python PPTX :)

**Comment by scanny** [comment]
We don't do bugs here :)

**Comment by scanny** [comment]
Should be fixed in release v0.6.13 later today.
